src/Avatar/index.tsx

```diff
@@ -1,20 +1,25 @@
 import { Avatar as AntdAvatar, AvatarProps as AntdAvatarProps } from 'antd';
 import React from 'react';
 
+import { Tooltip } from '../Tooltip';
 import { MLL_THEME } from '../theme';
 
 import { stringToHslaColor } from './utils';
 
 export const Avatar: typeof AntdAvatar = AntdAvatar;
 export type AvatarProps = AntdAvatarProps;
 
-export function UserAvatar(props: AvatarProps & { username: string }) {
-  const color = stringToHslaColor(props.username);
+type AvatarUserProps = { toolTipLabel?: string; initials: string };
+
+export function UserAvatar(props: AvatarProps & AvatarUserProps) {
+  const color = stringToHslaColor(props.initials);
   return (
-    <Avatar
-      style={{ backgroundColor: color, color: MLL_THEME.backgroundColor }}
-    >
-      {props.username}
-    </Avatar>
+    <Tooltip title={props.toolTipLabel ?? null}>
```

Comment:
Needless complection, composition works fine instead. Especially given the Tooltip is optional.

Reply:
that means you would prefer a new component that uses the label (UserAvatarWithLabel) and make the label property on this component obligatory?

Reply:
Clients can simply do this as needed:

```tsx
<Tooltip title="Foo"><UserAvatar /></Tooltip>
```

No need to increase the complexity of this component or introduce a new one.

Reply:
ok, I see